spec/requests/api/internal/v2/court_application_laa_references_spec.rb

```diff
@@ -39,7 +31,7 @@
       security [{ oAuth: [] }]
       parameter "$ref" => "#/components/parameters/transaction_id_header"
 
-      response(202, "Accepted") do
+      response(201, "Created") do
```

Comment:
This file is used to generate the swagger yaml, so you'll want to run `rails rswag` to update it.